test/system/post_show_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Post show page', type: :feature do
+  user = User.create(name: 'Abel', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
+  post = Post.create(author_id: user.id, title: 'Hello', text: 'This is my first post')
+  comment = Comment.create(user_id: user.id, post_id: post.id, text: 'text')
```

Comment:
- [ ] Your tests in this file failed to run because you are trying to create a comment with `user_id`, instead of `author_id`